server/src/uuid/model/taxonomy_term/mod.rs

```diff
@@ -236,3 +242,120 @@ impl TaxonomyTerm {
         typename.to_case(Case::Camel)
     }
 }
+
+#[derive(Error, Debug)]
+pub enum TaxonomyTermSetNameAndDescriptionError {
+    #[error(
+        "Taxonomy Term name and description cannot be set because of a database error: {inner:?}."
+    )]
+    DatabaseError { inner: sqlx::Error },
+    #[error(
+        "Taxonomy Term name and description cannot be set because of an event error: {inner:?}."
+    )]
+    EventError { inner: EventError },
+}
+
+impl From<sqlx::Error> for TaxonomyTermSetNameAndDescriptionError {
+    fn from(inner: sqlx::Error) -> Self {
+        Self::DatabaseError { inner }
+    }
+}
+
+impl From<EventError> for TaxonomyTermSetNameAndDescriptionError {
+    fn from(error: EventError) -> Self {
+        match error {
+            EventError::DatabaseError { inner } => inner.into(),
+            inner => Self::EventError { inner },
+        }
+    }
+}
+
+impl From<TaxonomyTermSetNameAndDescriptionError> for operation::Error {
+    fn from(error: TaxonomyTermSetNameAndDescriptionError) -> Self {
+        match error {
+            TaxonomyTermSetNameAndDescriptionError::DatabaseError {
+                inner: sqlx::Error::RowNotFound,
+            } => operation::Error::NotFoundError,
+            TaxonomyTermSetNameAndDescriptionError::DatabaseError { .. }
+            | TaxonomyTermSetNameAndDescriptionError::EventError { .. } => {
+                operation::Error::InternalServerError {
+                    error: Box::new(error),
+                }
+            }
+        }
+    }
+}
+
+impl TaxonomyTerm {
+    pub async fn set_name_and_description<'a, E>(
+        payload: &taxonomy_term_set_name_and_description_mutation::Payload,
+        executor: E,
+    ) -> Result<bool, TaxonomyTermSetNameAndDescriptionError>
+    where
+        E: Executor<'a>,
+    {
+        let mut transaction = executor.begin().await?;
+
+        let term_id = sqlx::query!(
+            r#"
+                SELECT term_id
+                    FROM term_taxonomy
+                    WHERE id = ?
+            "#,
+            payload.id
+        )
+        .fetch_one(&mut transaction)
+        .await?
+        .term_id as i32;
+
+        sqlx::query!(
+            r#"
+                UPDATE term
+                SET name = ?
+                WHERE id = ?
+            "#,
+            payload.name,
+            term_id,
+        )
+        .execute(&mut transaction)
+        .await?;
+
+        let mut description = "".to_string();
+
+        if payload.description.is_some() {
+            description = payload.description.clone().unwrap();
+        }
```

Comment:
`description = payload.unwrap_or("".to_string())

Reply:
instead of `payload.description.clone().unwrap_or("".to_string())`, clippy suggested to use `unwrap_or_else`. 
Done in 5ae0028